src/Polar/second_order.jl

```diff
@@ -404,13 +404,13 @@ function AutoDiff.partials!(hess::ArrowheadHessian)
     n = length(duals)
     duals_ = reshape(reinterpret(T, duals), N+1, n)
 
-    if isa(H, SparseMatrixCSC)
+    if !_iscsr(H)
```

Comment:
I would prefer having two functions `iscsr` and `iscsc` and write the condition as 
```julia
if iscsc(H)
    baba
elseif iscsr(H)
    abab
else 
    error("Not defined")
end
```
what do you think?

Reply:
Done.